provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitionerrole/PractitionerRoleController.java

```diff
@@ -49,69 +53,49 @@
 
   private PractitionerRole.Bundle bundle(
       MultiValueMap<String, String> parameters, int page, int count) {
-    PractitionerRoleWrapper root = search(parameters);
+    Pair<List<PractitionerRoleWrapper>, Integer> root = search(parameters);
     LinkConfig linkConfig =
         LinkConfig.builder()
             .path("PractitionerRole")
             .queryParams(parameters)
             .page(page)
             .recordsPerPage(count)
-            .totalRecords(1)
+            .totalRecords(root.getRight())
             .build();
     return bundler.bundle(
         BundleContext.of(
             linkConfig,
-            Collections.singletonList(root),
+            root.getLeft(),
             transformer,
             PractitionerRole.Entry::new,
             PractitionerRole.Bundle::new));
   }
 
+  /** Builds the PractitionerWrapper and returns it. */
+  private PractitionerRoleWrapper practitionerRoleWrapperBuilder(
+      ProviderContactsResponse providerContactsResponse,
+      ProviderResponse.Value providerResponse,
+      ProviderSpecialtiesResponse providerSpecialtiesResponse) {
+
+    return new PractitionerRoleWrapper.PractitionerRoleWrapperBuilder()
```

Comment:
You can also do `PractitionerRoleWrapper.builder()` as a more concise way to create the builder.